thoth-app/src/models/contributor/mod.rs

```diff
@@ -55,11 +58,26 @@ impl Contributor {
                 <td>{&self.contributor_id}</td>
                 <td>{&self.full_name}</td>
                 <td>{orcid}</td>
+                <td>{&self.updated_at.format("%F %T")}</td>
             </tr>
         }
     }
 }
 
+impl Default for Contributor {
+    fn default() -> Contributor {
+        Contributor {
+            contributor_id: "".to_string(),
```

Comment:
just fyi (no need to change anything), you can call `Default:::default()` on these fields individually, e.g. `contributor_id: Default::default()`, as strings and options do implement Default